src/cloud_what/providers/aws.py

```diff
@@ -24,13 +24,19 @@
 import time
 import os
 
-from typing import Union
+from typing import Union, Any
 
 from cloud_what._base_provider import BaseCloudProvider
 
 
 log = logging.getLogger(__name__)
 
+# Instance from one region will be redirected to another region's CDS for content
+REDIRECT_MAP = {
+    'us-gov-west-1': 'us-west-2',
```

Comment:
How static/dynamic is this mapping? Will we have to update this over time?

Reply:
It is quite static. I will tell you more details on IRC.

Reply:
Great! Thank you!